httpbakery/client.go

```diff
@@ -333,11 +333,12 @@ func NewCookie(ms macaroon.Slice) (*http.Cookie, error) {
 	if err != nil {
 		return nil, errgo.Notef(err, "cannot marshal macaroons")
 	}
-	return &http.Cookie{
+	cookie := &http.Cookie{
 		Name:  fmt.Sprintf("macaroon-%x", ms[0].Signature()),
 		Value: base64.StdEncoding.EncodeToString(data),
-		// TODO(rog) other fields, particularly expiry time.
```

Comment:
The todo could be left for "other fields" though (without notice on expiry time).